docs/core/proto-docs.md

```diff
@@ -7595,6 +7598,11 @@ SignMode represents a signing mode with its own security guarantees.
 | SIGN_MODE_DIRECT | 1 | SIGN_MODE_DIRECT specifies a signing mode which uses SignDoc and is verified with raw bytes from Tx |
 | SIGN_MODE_TEXTUAL | 2 | SIGN_MODE_TEXTUAL is a future signing mode that will verify some human-readable textual representation on top of the binary representation from SIGN_MODE_DIRECT |
 | SIGN_MODE_LEGACY_AMINO_JSON | 127 | SIGN_MODE_LEGACY_AMINO_JSON is a backwards compatibility mode which uses Amino JSON and will be removed in the future |
+| SIGN_MODE_EIP_191 | 191 | SIGN_MODE_EIP_191 specifies the sign mode for EIP 191 signing on the Cosmos SDK. Ref: https://eips.ethereum.org/EIPS/eip-191
+
+Currently, SIGN_MODE_EIP_191 is registered as a SignMode enum variant, but is not implemented on the SDK by default. To enable EIP-191, you need to pass a custom `TxConfig` that has an implementation of `SignModeHandler` for EIP-191. The SDK may decide to fully support EIP-191 in the future.
```

Comment:
I wonder where this `SIGN_MODE_EIP_191` was found.
Because the documentation for cosmos-sdk version 0.45.3 doesn't include this. :)

Reply:
You're right. It is included in v0.45.5. It seems that it's tangled when checking out branches.
BTW, I'm preparing new PR for v0.45.4. Because the test of resetta is failed due to the version of go, which is required 1.17 or upper ver. It was fixed in v0.45.4.. After I test locally, I'll make a new PR.
Thanks for comments!

Reply:
Aha. OK.